ui/src/components/containers/ContainerInspect.js

```diff
@@ -11,7 +11,10 @@ import DateTime from 'react-datetime';
 import '../../../node_modules/react-datetime/css/react-datetime.css';
 
 import { logsContainer, execContainer, topContainer } from '../../api';
-import { shortenImageName } from '../../lib';
+import { shortenImageName, showError, showSuccess } from '../../lib';
```

Comment:
Is `shortenImageName` being used here?  As an aside, it might be worth making sure that `eslint` output is available in your editor (https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#displaying-lint-output-in-the-editor).